content/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -286,7 +286,7 @@ The Kubernetes API serves and handles the storage of your custom resource.
 The name of a CRD object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 -->
-## CustomResourceDefinitions
+## 定制资源定义 {#customresourcedefinitions}
```

Comment:
Don't do that!!! "CustomResourceDefinition" in its current spelling means a resource name in the API. Not supposed to be translated.